### DISCUSSION TITLE
When ingress_controller = "none" a Traefik instance is still deployed and running on nodes

### DISCUSSION THREAD

**Discussion by laurentpellegrino** [kube-hetzner/terraform-hcloud-kube-hetzner#1383] [Ideas]
### Description

I am expecting no Traefik instances to run.

### Kube.tf file

```terraform
allow_scheduling_on_control_plane = true
  cluster_name = "my-cluster"
  enable_klipper_metal_lb = "true"
  ingress_controller = "none"
  restrict_outbound_traffic = false
```

### Screenshots

![Screenshot from 2024-01-13 21-50-27](https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/assets/128898/1b0d39ca-6d29-4bea-a7fe-2708e23a7f66)

### Platform

Linux

**Comment by mysticaltech** [comment]
@lpellegr This setting does not apply retroactive fashion, it needs to be setup from the get go, otherwise you can use kubectl get crd, find the name of the rancher helm charts, list them, and delete them to remove traefik.

**Comment by mysticaltech** [comment]
However we will try to make it apply retroactively, it's an important functionality to have.

**Comment by maximdogonov** [comment]
Today I replace ingress from traefik to nginx, nginx deployed but traefik not removed

**Comment by mysticaltech** [comment]
Yes @maximdogonov, you have to:

kubectl get crds | grep helm
kubectl get ... -A
kubectl delete helmxxxx traefik

Something along these lines.
